### ISSUE TITLE
/sethome

### ISSUE THREAD

**Issue by nthirtyone** [nthirtyone/avorion-scripts#8]
```
could not execute function 'initialize' in '"data/scripts/player/cmd/sethome.lua"':
data/scripts/player/cmd/sethome.lua:28: attempt to perform arithmetic on a string value
stack traceback:
        data/scripts/player/cmd/sethome.lua:28: in function <data/scripts/player/cmd/sethome.lua:3>

Setting script "data/scripts/player/cmd/sethome.lua" to invalid.
```
Original post from forums: [here](http://www.avorion.net/forum/index.php/topic,830.msg5672.html#msg5672).
It was noted on @wmark's version of script.

Changes on `new-commands` (yee, let's put it all together into a pile of... things).

**Comment by mark-kubacki** [comment]
You have reverted your version of `/sethome` (which I don't mind), and I am using mine as part of my ACI and Docker images from here (see folder *contrib*):
https://github.com/wmark/aci-avorion-server

If a script is not shared by us, I'd like to entertain bug reports only if issued for the applicable repository.

Nonetheless, `Faction()` doesn't seem to return what it previously returned, or its parameters have changed. I will look into it, but this issue can be closed here.

**Comment by nthirtyone** [comment]
Actually, it was the same version of the script as your repository, but, yeah, not `contrib`. It seems I got tricked by my own perception. But anyway, I think it doesn't matter anymore. I have changed `/sethome` a little bit more to allow players to change their home sector more widely: it allows them to do that in any sector with own or friendly station (`new-commands` branch).

I'm still not sure about "Resistance Outpost", should it be added?

Oh, and I finally managed to allow you to modify repository directly, sorry it took so long.
